$MirOS: ports/sysutils/e2fsprogs/patches/patch-debugfs_debugfs_c,v 1.1.7.1 2005/03/18 15:52:04 tg Exp $
--- debugfs/debugfs.c.orig	Fri May  6 13:04:14 2005
+++ debugfs/debugfs.c	Tue Jan  3 02:52:56 2006
@@ -195,16 +195,16 @@ static void close_filesystem(NOARGS)
 	if (current_fs->flags & EXT2_FLAG_IB_DIRTY) {
 		retval = ext2fs_write_inode_bitmap(current_fs);
 		if (retval)
-			com_err("ext2fs_write_inode_bitmap", retval, "");
+			com_err("ext2fs_write_inode_bitmap", retval, NULL);
 	}
 	if (current_fs->flags & EXT2_FLAG_BB_DIRTY) {
 		retval = ext2fs_write_block_bitmap(current_fs);
 		if (retval)
-			com_err("ext2fs_write_block_bitmap", retval, "");
+			com_err("ext2fs_write_block_bitmap", retval, NULL);
 	}
 	retval = ext2fs_close(current_fs);
 	if (retval)
-		com_err("ext2fs_close", retval, "");
+		com_err("ext2fs_close", retval, NULL);
 	current_fs = NULL;
 	return;
 }
@@ -631,7 +631,7 @@ void do_chroot(int argc, char *argv[])
 
 	retval = ext2fs_check_directory(current_fs, inode);
 	if (retval)  {
-		com_err(argv[1], retval, "");
+		com_err(argv[1], retval, NULL);
 		return;
 	}
 	root = inode;
@@ -902,7 +902,7 @@ void do_change_working_dir(int argc, cha
 
 	retval = ext2fs_check_directory(current_fs, inode);
 	if (retval) {
-		com_err(argv[1], retval, "");
+		com_err(argv[1], retval, NULL);
 		return;
 	}
 	cwd = inode;
@@ -1013,7 +1013,7 @@ static void make_link(char *sourcename, 
 	retval = ext2fs_link(current_fs, dir, dest, ino, 
 			     ext2_file_type(inode.i_mode));
 	if (retval)
-		com_err("make_link", retval, "");
+		com_err("make_link", retval, NULL);
 	return;
 }
 
@@ -1096,7 +1096,7 @@ static void unlink_file_by_name(char *fi
 	}
 	retval = ext2fs_unlink(current_fs, dir, basename, 0, 0);
 	if (retval)
-		com_err("unlink_file_by_name", retval, "");
+		com_err("unlink_file_by_name", retval, NULL);
 	return;
 }
 
@@ -1148,7 +1148,7 @@ void do_find_free_block(int argc, char *
 		retval = ext2fs_new_block(current_fs, free_blk + 1, 0,
 					  &free_blk);
 		if (retval) {
-			com_err("ext2fs_new_block", retval, "");
+			com_err("ext2fs_new_block", retval, NULL);
 			return;
 		} else
 			printf("%d ", free_blk);
@@ -1190,7 +1190,7 @@ void do_find_free_inode(int argc, char *
 
 	retval = ext2fs_new_inode(current_fs, dir, mode, 0, &free_inode);
 	if (retval)
-		com_err("ext2fs_new_inode", retval, "");
+		com_err("ext2fs_new_inode", retval, NULL);
 	else
 		printf("Free inode found: %u\n", free_inode);
 }
@@ -1251,11 +1251,11 @@ void do_write(int argc, char *argv[])
 
 	fd = open(argv[1], O_RDONLY);
 	if (fd < 0) {
-		com_err(argv[1], errno, "");
+		com_err(argv[1], errno, NULL);
 		return;
 	}
 	if (fstat(fd, &statbuf) < 0) {
-		com_err(argv[1], errno, "");
+		com_err(argv[1], errno, NULL);
 		close(fd);
 		return;
 	}
@@ -1269,7 +1269,7 @@ void do_write(int argc, char *argv[])
 
 	retval = ext2fs_new_inode(current_fs, cwd, 010755, 0, &newfile);
 	if (retval) {
-		com_err(argv[0], retval, "");
+		com_err(argv[0], retval, NULL);
 		close(fd);
 		return;
 	}
@@ -1286,7 +1286,7 @@ void do_write(int argc, char *argv[])
 				     EXT2_FT_REG_FILE);
 	}
 	if (retval) {
-		com_err(argv[2], retval, "");
+		com_err(argv[2], retval, NULL);
 		close(fd);
 		return;
 	}
@@ -1305,7 +1305,7 @@ void do_write(int argc, char *argv[])
 	if (LINUX_S_ISREG(inode.i_mode)) {
 		retval = copy_file(fd, newfile);
 		if (retval)
-			com_err("copy_file", retval, "");
+			com_err("copy_file", retval, NULL);
 	}
 	close(fd);
 }
@@ -1358,7 +1358,7 @@ void do_mknod(int argc, char *argv[])
 		return;
 	retval = ext2fs_new_inode(current_fs, cwd, 010755, 0, &newfile);
 	if (retval) {
-		com_err(argv[0], retval, "");
+		com_err(argv[0], retval, NULL);
 		return;
 	}
 	printf("Allocated inode: %u\n", newfile);
@@ -1373,7 +1373,7 @@ void do_mknod(int argc, char *argv[])
 				     filetype);
 	}
 	if (retval) {
-		com_err(argv[1], retval, "");
+		com_err(argv[1], retval, NULL);
 		return;
 	}
         if (ext2fs_test_inode_bitmap(current_fs->inode_map,newfile))
@@ -1411,7 +1411,7 @@ void do_mkdir(int argc, char *argv[])
 		*cp = 0;
 		parent = string_to_inode(argv[1]);
 		if (!parent) {
-			com_err(argv[1], ENOENT, "");
+			com_err(argv[1], ENOENT, NULL);
 			return;
 		}
 		name = cp+1;
@@ -1431,7 +1431,7 @@ try_again:
 		goto try_again;
 	}
 	if (retval) {
-		com_err("ext2fs_mkdir", retval, "");
+		com_err("ext2fs_mkdir", retval, NULL);
 		return;
 	}
 
@@ -1617,7 +1617,7 @@ void do_expand_dir(int argc, char *argv[
 
 	retval = ext2fs_expand_dir(current_fs, inode);
 	if (retval)
-		com_err("ext2fs_expand_dir", retval, "");
+		com_err("ext2fs_expand_dir", retval, NULL);
 	return;
 }
 
